exo/inference/tinygrad/inference.py

```diff
@@ -27,7 +27,7 @@
 }
 
 
-def build_transformer(model_path: Path, shard: Shard, model_size="8B", device=None):
+def build_llama(model_path: Path, shard: Shard, model_size="8B", device=None):
```

Comment:
why change this to `build_llama`?

Reply:
Every model will have its own build function. So build_llava, build_llama, build_mamba, etc...

Reply:
ah, i missed that you changed it below. These AsyncIO calls are not quite as reader friendly 😄